nss-day-cohort-41/tabloid-fullstack-shocking-secret-keepers
# Bs reaction

# Description
Completed Add Post Reaction and view post reactions on Post Details 
 
Fixes #35  #36

## Type of change
Please delete options that are not relevant.

# Testing Instructions

Load Tabloid from remote repository
Run ReactionSeed.sql file in SQL folder

Npm start/launch Tabloid
Login 

Post Reaction - View existing reactions
goto route: "/post/details/1"
Previously added reaction images will be visible at the bottom of the post
- a count of each time a reaction type has been selected by users for that post is given on the right of the image
 
Post Reaction Add & Post Reaction List
Click on any "Post" from navbar
Click on any post "Details” button
View post details and click on "Add Reaction" Button on bottom of post
Once a reaction is chosen the "Add Reaction" options should disappear and your reaction should be added to reactions on the post.
- if your reaction was already on the post, you will see the reaction number change.
- if your reaction type is new to the post, you will see the reaction image added with a reaction number of "1"

- reactions can not be removed
- only 1 reaction from a user can be added per post

# Checklist:
